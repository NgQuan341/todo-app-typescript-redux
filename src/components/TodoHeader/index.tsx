import React from 'react'
import { Typography } from 'antd'

const TodoHeader = () => {
  const { Title } = Typography
  return (
    <>
    <Title style={{ textAlign: 'center' }} level={2}>TODO APP</Title>
    </>
  )
}

export default TodoHeader