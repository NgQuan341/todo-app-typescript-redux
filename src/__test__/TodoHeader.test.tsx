
import { shallow, mount } from 'enzyme'
import TodoHeader from '../components/TodoHeader';
import { Typography } from 'antd';

const setupShallow = () => {
   let wrapper = shallow<typeof TodoHeader>(<TodoHeader />)
   return wrapper
}
const setupMount = () => {
    let wrapper = mount<typeof TodoHeader>(<TodoHeader />)
    return wrapper
 }
 
describe('TodoHeader component', () =>{
    test('renders without error', () => {
        const wrapper = setupShallow()                 
        expect(wrapper.html()).toMatchSnapshot()
    })
    // test ('test text in component Typography', () =>{
    //     const wrapper = setupShallow()        
    //     // expect(wrapper.find(<Typography/>)).text().toEqual('TODO APP')
        
    // })
})