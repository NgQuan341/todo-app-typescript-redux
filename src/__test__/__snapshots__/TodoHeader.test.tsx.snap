// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoHeader component renders without error 1`] = `"<h2 class=\\"ant-typography\\" style=\\"text-align:center\\">TODO APP</h2>"`;
